neuromation/cli/job.py

```diff
@@ -393,25 +400,69 @@ def job() -> None:
     neuro job port-forward my-job- 2080:80 2222:22 2000:100
 
     """
-    job_id = await resolve_job(job, client=root.client)
-    async with AsyncExitStack() as stack:
-        for local_port, job_port in local_remote_port:
-            click.echo(
-                f"Port localhost:{local_port} will be forwarded "
-                f"to port {job_port} of {job_id}"
-            )
-            await stack.enter_async_context(
-                root.client.jobs.port_forward(
-                    job_id, local_port, job_port, no_key_check=no_key_check
-                )
-            )
 
-        click.echo("Press ^C to stop forwarding")
-        try:
-            while True:
-                await asyncio.sleep(1)
-        except KeyboardInterrupt:
-            pass
+    from asyncio import Task
+
+    _tasks: List[Task] = []
+    _stopped: bool = False
+
+    def _done(local_remote_port: Tuple[int, int], task: Task):
```

Comment:
`done()` could be defined as local function inside `forward()`, so you would not need to pass `local_remote_port` and use `partial()`, because it is available as nonlocal.